### PR TITLE
Add script and Dockerfile

### PR DESCRIPTION
**Pull request by katieclaiborne-duet**
## [Airtable record](https://airtable.com/applRnjY08XXmLq3A/tbl9NslAfxuW3Etym/viwQQeE8Mihy1eBXk/recDJalFiEaMxy4zy?blocks=hide)

Relates to https://github.com/Duet-Technologies/terraform/pull/15

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM python:3.12-slim
+COPY . .
```
Comment (katieclaiborne-duet) [inline Dockerfile:2]: What does this line do?

File: export.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from google.cloud import bigquery
+
+client = bigquery.Client()
+bucket_name = "duet-point-click-care"
+bucket_path = "outbound"
```
Comment (katieclaiborne-duet) [inline export.py:5]: We may want to add another level - maybe date?

File: export.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+project    = "vital-ensign-388617"
+dataset_id = "dbt_scratch"
+table_id   = "xpt_pcc_eligibility"
+
+destination_uri = "gs://{}/{}/{}".format(bucket_name, bucket_path, "duet_eligibility.csv")
```
Comment (katieclaiborne-duet) [inline export.py:11]: Should we guarantee that the file name will be unique? Maybe add timestamp?

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+RUN pip install google-cloud-bigquery
+ENTRYPOINT ["python", "export.py"]
```
Comment (katieclaiborne-duet) [inline Dockerfile:4]: Is it worth adding a Dockerfile at this point, or should we wait to orchestrate the export and push jobs together?

Comment (JoeVieira) [reply Dockerfile:4]: I think that we'd want to use cloud function for this ( we don't need to manage the container, etc )

**Review by JoeVieira — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 4 above)
Comment (katieclaiborne-duet) [reply Dockerfile:4]: Makes sense! I'll work on that today.

**Comment by katieclaiborne-duet** [comment]
Closing, will reopen in broader "data exchange" repo.

**Pull request closed without merging**